ci/pipeline.yml

```diff
@@ -0,0 +1,216 @@
+#@ load("@ytt:data", "data")
+
+#@ def pipeline_image():
+#@   return data.values.docker_registry + "/galoy-app-pipeline"
```

Comment:
Need to consolidate the various `pipeline-image`s in concourse-shared... (just a reminder)